spec/collections/samples/sample_ontology_link.yaml

```diff
@@ -0,0 +1,47 @@
+name: sample_ontology_link
```

Comment:
Probably ask @scanon whether these should be "time travel" collections. If it's used in a narrative that should be (theoretically) reproducible, then probably they will want them to be time travel-able.

If Shane does want this, I'll tell you how to do the time travel style collections (it's not very different)